src/Components/MainTableView/DataStorageClasses/SecondaryTableAttribute.tsx

```diff
@@ -46,9 +63,14 @@ class SecondaryTableAttribute extends TableAttribute {
     )
   }
 
-  static getAttributeInputBlock(secondaryTableAttribute: SecondaryTableAttribute, currentValue: any, handleChange: any) {
+  /**
+   * Construct input block based on secondaryTableAttribute by extracting it type and other information
+   * @param secondaryTableAttribute 
+   * @param currentValue urrentValue of the input block for binding
+   * @param handleChange Call back function for when the value of the input block changes
+   * @returns 
+   */
+  static getSecondaryAttributeInputBlock(secondaryTableAttribute: SecondaryTableAttribute, currentValue: any, handleChange: (event: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLSelectElement>, attributeName: string) => void) {
```

Comment:
Here is one `any` that was missed.

Reply:
That one I left it as any cause that requires mutiple types and heavily depends on what the input block type is. I could make it more specific but it is quite a bit of types to account for and will probably introduced bugs if I miss one.

Reply:
@Synicix I see, fair enough. I would say then we can keep this as `any` but let's add a note to the docs to indicate why `any` was chosen here. In the docs for that argument, you could say something like

`@param currentValue CurrentValue of the input block for binding. Type any used here as there are many possible types with all the available input blocks.`

Reply:
Fair enough

Reply:
There, I updated the docs